Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement.py

```diff
@@ -538,9 +688,19 @@ def main() -> None:
         elif command == 'azure-sql-generate-login-url':
             return_results(generate_login_url(client.ms_client))
 
+        elif command == 'azure-sql-subscriptions-list':
+            return_results(subscriptions_list_command(client))
+
+        elif command == 'azure-sql-resource-group-list':
+            return_results(resource_group_list_command(client, args, subscription_id))
+
     # Log exceptions and return errors
     except Exception as e:
-        return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
+        if 'Error in API call [404] - Not Found' in e.message:
```

Comment:
I rather this to be in the def http_request() function.
Check what returns from the response and then do this check.
If status_code == 404 and 'Not Found':
return message.